src/ray_tracer.cpp

```diff
@@ -117,24 +117,26 @@ simulateRangeScan (const nm::OccupancyGrid& grid, const gm::Pose& sensor_pose,
   {
     const double theta = scanner_info.angle_min+i*scanner_info.angle_increment;
     const gm::Point scan_max =
-      rayEndPoint(p0, theta0 + theta, scanner_info.range_max+1);
+      rayEndPoint(p0, theta0 + theta, scanner_info.range_max + grid.info.resolution);
```

Comment:
Looking at https://github.com/clearpathrobotics/occupancy_grid_utils/blob/a70b91473506f924167e3c945d790a8fa692fb62/src/ray_tracer.cpp#L43, I think it's probably enough with `scanner_info.range_max`, but I'm also OK with adding the `grid.info.resolution`.